runtime/vm/jvminit.c

```diff
@@ -3561,7 +3568,24 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		IDATA enableCRIU = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXENABLECRIU, NULL);
 		IDATA disableCRIU = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXDISABLECRIU, NULL);
 		if (enableCRIU > disableCRIU) {
-			vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_CRIU_SUPPORT;
+			PORT_ACCESS_FROM_JAVAVM(vm);
+			vm->checkpointState = (J9CRIUCheckpointState*) j9mem_allocate_memory(sizeof(J9CRIUCheckpointState), OMRMEM_CATEGORY_VM);
+			memset(vm->checkpointState, 0, sizeof(J9CRIUCheckpointState));
+
+			if (NULL == vm->checkpointState) {
```

Comment:
@tajila This test needs to be before the call to `memset()` (which would fail if true).